paseto/README.md

```
@@ -246,32 +265,68 @@ func restricted(c fiber.Ctx) error {
-#### Test it
+### Custom Extractor
 
-_Login using username and password to retrieve a token._
+```go
+package main
 
-```
-curl --data "user=john&pass=doe" http://localhost:8088/login
-```
+import (
+	"time"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)